web-client/integration-tests/docketClerkVerifiesConsolidatedCases.test.js

```diff
@@ -105,4 +106,13 @@ describe('Docket Clerk verifies Consolidated Cases', () => {
 
   loginAs(cerebralTest, 'docketclerk@example.com');
   docketClerkVerifiesConsolidatesCases(cerebralTest);
+
+  const updatedCaseCaption = 'I am the new case caption!';
+  updateACaseCaption(cerebralTest, updatedCaseCaption);
+  it('docketClerk updates Case Caption', async () => {
+    const caseDetail = cerebralTest.getState('caseDetail');
+    expect(caseDetail.caseCaption).toBeDefined();
+    expect(caseDetail.caseCaption).toEqual(updatedCaseCaption);
+  });
+  docketClerkVerifiesConsolidatesCases(cerebralTest);
```

Comment:
:pushpin: I think the name of this function should be `docketClerkVerifiesConsolidatedCases`, but I know that file wasn't created in this PR. Maybe something that can be corrected at some point. Or maybe I'm missing the point of the file.